test/test_jit.py

```diff
@@ -14463,6 +14463,31 @@ def forward(self, a):
         output = m_loaded(input)
         self.assertEqual(3 * input, output)
 
+    def test_overloaded_fn(self):
```

Comment:
It'll get long but I think it should have a test every magic method to make sure the switch in compiler.cpp is correct

Reply:
sure 🌵